### PR TITLE
Update storage README.md

### PR DESCRIPTION
**Pull request by syndbg**
Add missing option how to use the storage client with an API key.
Update button names to refer to the actual names in the UI.

### CODE REVIEW

**Comment by syndbg** [comment]
I signed it!

File: packages/storage/README.md [hunk 1]
```diff
@@ -90,20 +90,26 @@ If you are not running this client on Google Compute Engine, you need a Google D
   * Google Cloud Storage
   * Google Cloud Storage JSON API
 4. Navigate to **APIs & auth** >  **Credentials** and then:
-  * If you want to use a new service account, click on **Create new Client ID** and select **Service account**. After the account is created, you will be prompted to download the JSON key file that the library uses to authenticate your requests.
-  * If you want to generate a new key for an existing service account, click on **Generate new JSON key** and download the JSON key file.
+  * If you want to use a new service account key, click on **Create credentials** and select **Service account key**. After the account key is created, you will be prompted to download the JSON key file that the library uses to authenticate your requests.
+  * If you want to generate a new service account key for an existing service account, click on **Generate new JSON key** and download the JSON key file.
+  * If you want to use a new API key, click on **Create credentials** and select **API key**. After the API key is created, you will see a newly opened modal with the API key in a field named **Your API key** that the library uses to authenticate your requests.
```
Comment (stephenplusplus) [inline packages/storage/README.md]: For any of our APIs that accept an API key, click on **Create credentials**...

File: packages/storage/README.md [hunk 2]
```diff
@@ -90,20 +90,26 @@ If you are not running this client on Google Compute Engine, you need a Google D
...
+  * If you want to generate a new API key for an existing API key, click on an existing API key and click **Regenerate key**.
 
 ``` js
 var projectId = process.env.GCLOUD_PROJECT; // E.g. 'grape-spaceship-123'
 
 var gcs = require('@google-cloud/storage')({
+  // 1. Specify projectId
```
Comment (stephenplusplus) [inline packages/storage/README.md]: I don't think this or the changes below are necessary.

Comment (syndbg) [inline packages/storage/README.md]: Imo it's a clarification that there are two approaches to the authentication.

However I agree that the changes below seem unnecessary cause they're probably not making things better.

**Comment by syndbg** [comment]
```
Thank you very much for this! I believe we only support an API key for the Translate API, so it is likely misleading to suggest it's always an option.

Instead of making these changes only in packages/storage/README.md, it would be amazing if you did them all of the README files (packages/*/README.md) including the root readme (README.md).
```

Sure. I can change them in this PR if you'd like.

File: packages/storage/README.md (hunk 1 above)
Comment (stephenplusplus) [inline packages/storage/README.md]: Woops, sorry. I meant to replace the line:

```diff
- If you want to use a new API key, click on **Create credentials**
+ For any of our APIs that accept an API key, click on **Create credentials**
```

File: packages/storage/README.md (hunk 2 above)
Comment (stephenplusplus) [inline packages/storage/README.md]: Yeah, it's not that it's unhelpful, I just think we have it covered already.

**Comment by stephenplusplus** [comment]
> Sure. I can change them in this PR if you'd like.

Please, that would be great!

**Comment by stephenplusplus** [comment]
Thanks again for this @syndbg, I've put this into a new PR to save you from more work. You've done enough! 

New PR: #1791

**Pull request closed without merging**